### DISCUSSION TITLE
Fatture elettroniche fornitore in acquisti

### DISCUSSION THREAD

**Discussion by donlegro** [fattureincloud/api#104] [Q&A] [answered]
Salve `received_documents` legge le fatture presenti in Acquisti.
Quindi per vedere le fatture elettroniche ricevute c'è bisogno che qualcuno entri nel portale Fatture in cloud e registri queste fatture arrivate in Acquisti.
C'è un'impostazione o un comando o una chiamata API per fare questo esternamente o automaticamente. Senza fare il login tutte le volte?
Grazie.

**Comment by emajo** [comment, marked as answer]
Ciao @donlegro, purtroppo questa funzionalità non è presente, è neccessario registrare manualmente le fatture elettroniche ricevute.
